packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1059,6 +1069,38 @@ contract VotingV2 is
         return spamDeletionProposals[spamDeletionRequestId];
     }
 
+    function signalOnEmergencyAction(
+        bytes32 identifier,
+        uint256 actionTime,
+        bytes memory ancillaryData
+    ) public {
+        EmergencyAction storage action = emergencyActions[_encodePriceRequest(identifier, actionTime, ancillaryData)];
```

Comment:
Maybe we could have a `require(getCurrentTime()>deploymentTimestamp+30days)` to prevent 51% attacks at the beginning

Reply:
ye, this is a great point. can add something like this. it does mean though that if we break in the 30 days we cant use this and will be stuck until this require has passed. we could make it like 10 days or something such that we've got enough deposits to keep things functional.